cmd/gscheduler/cmd/gscheduler.go

```diff
@@ -0,0 +1,110 @@
+/**
```

Comment:
It looks almost the same logic as shutdown_instance.go in Root.
Could you not unify the code?